modules/configuring-albo-on-sts-cluster-predefined-credentials.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+// * networking/installing-albo-sts-cluster.adoc
+
+:_content-type: PROCEDURE
+[id="nw-installing-albo-on-sts-cluster-predefined-credentials_{context}"]
+= Configuring the AWS Load Balancer Operator on Secure Token Service cluster by using predefined credentials
+
+You can use the predefined `CredentialsRequest` object by specifying the `spec.credential` field in the AWS load Balancer Controller custom resource (CR).
+
+.Prerequisites
+
+* You must extract and prepare the `ccoctl` binary.
```

Comment:
IMO, since this is part of a prerequisites section, we must keep it at the beginning of each chapter. But I can check once the peer-review process begins if we can eliminate the repetitive commands